### ISSUE TITLE
[a11y] Cannot distinguish if cells are read-only between views

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1232]
**Describe the bug**
The background colors and read-only indicators are different between forms and pop-outs.

**To Reproduce**
1. View a CE record from the CO form
2. View the same CE record on its own page

**Screenshots**
Full form view:
<img width="940" alt="image" src="https://user-images.githubusercontent.com/37256050/163510681-44a8e719-07e2-4d77-a0a2-133876b4790d.png">

Subview:
<img width="846" alt="image" src="https://user-images.githubusercontent.com/37256050/163510699-7ee05e7b-14e7-41d7-991d-40c588fa9463.png">

The gray background is an editable field in the subview pop-out:
<img width="324" alt="image" src="https://user-images.githubusercontent.com/37256050/163510722-2442f4a1-6813-4c90-aaed-3822c7596f29.png">

The white background is an editable field on the form:
<img width="350" alt="image" src="https://user-images.githubusercontent.com/37256050/163510784-8c24910b-761a-4d01-a632-58dc0e81ca62.png">

**Comment by maxxxxxdlp** [comment]
Fixed in 4ac922a8
